CHANGELOG.asciidoc

```diff
@@ -89,7 +89,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 *Affecting all Beats*
 
 - Unify dashboard exporter tools. {pull}9097[9097]
-- Use _doc as document type of the Elasticsearch major version is 7. {pull}9056[9056]
+- Use _doc as document type. {pull}9056[9056]{pull}9573[9573]
```

Comment:
It could be argued this is a breaking change as it now also applies to ES 6?

Reply:
moved to breacking changes.